apps/autorelayer-interop/package.json

```diff
@@ -1,13 +1,29 @@
 {
   "name": "@eth-optimism/autorelayer-interop",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum-optimism/ecosystem.git",
+    "directory": "apps/autorelayer-interop"
+  },
+  "homepage": "https://github.com/ethereum-optimism/ecosystem/tree/main/apps/autorelayer-interop#readme",
+  "bugs": {
+    "url": "https://github.com/ethereum-optimism/ecosystem/issues"
+  },
   "version": "0.0.2",
-  "private": true,
   "type": "module",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
```

Comment:
can export the app here instead of index.ts

Reply:
fixed